src/Microsoft.AspNetCore.Routing/RouteEndpoint.cs

```diff
@@ -59,5 +59,14 @@ public sealed class RouteEndpoint : Endpoint
         /// Gets the <see cref="RoutePattern"/> associated with the endpoint.
         /// </summary>
         public RoutePattern RoutePattern { get; }
+
+        internal IReadOnlyDictionary<string, object> ResolveRouteValues()
```

Comment:
If we're planning on removing `IRouteValuesAddressMetadata` can we just leave this ugly for now? This shouldn't be used anywhere besides the RVAS

Reply:
Is IRouteValuesAddressMetadata being removed or made obsolete?
And if it is obsolete, is it being ignored all together, or used as a fallback source of required values?